README.md

```diff
@@ -1,13 +1,55 @@
-There are a lot of different ways to organize a workshop, but if you've got the [Minimum Viable Workshop](/Minimum-Viable-Workshop.md), you'll have a great time. There are way more people in the world who are excited to teach and learn Clojure than you know.
+# Get involved
 
-The [Workshop Planning Timeline](Workshop-Planning-Timeline.md) is a good place to start; it's divided into sections before, during, and after the workshop. There are *a lot* of individual tasks, so we recommend organizing in a team.
+ClojureBridge aims to increase diversity within the Clojure community by offering free, beginner-friendly Clojure programming workshops for women.
 
-We have past organizers who can help you figure out what to do. Email info@clojurebridge.org and we'll introduce you to an organizing mentor!
+ClojureBridge was started in 2014 and is part of [Bridge Foundry](http://bridgefoundry.org/). All around the world there have been workshops organized by people like you! Many people volunteered in different ways. Here are some insights for you to get involved:

```

Comment:
2014 sounds right to me, but @bridgethillyer is the source of truth on this matter. :)